src/test/java/RouterTest.java

```diff
@@ -0,0 +1,53 @@
+import java.util.HashMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RouterTest {
+  private HashMap<String, Handler> routes; 
+  private Router router;
+  
+  @Before 
+  public void setup() {
+    this.routes = new HashMap<String, Handler>();
+    String rootPath = System.getProperty("user.dir");
+    String jarDirectoryPath = rootPath + "/build/libs";
+    this.routes.put("/", new RootHandler(jarDirectoryPath));
+
+    this.router = new Router(routes);
+  }
+  
+  @Test 
+  public void returnsNotFoundHandlerWhenURIIsNotFound() {
```

Comment:
test name doesn't quite match what you're testing - rename to `returnsNotFoundWhenURIIsNotFound`